inception-concept-linking/src/main/java/de/tudarmstadt/ukp/inception/conceptlinking/util/FileUtils.java

```diff
@@ -42,7 +42,7 @@
 public class FileUtils
 {
 
-    private final static Logger log = LoggerFactory.getLogger(FileUtils.class);
```

Comment:
You mix two type of refactorings. One is about naming conventions. The other is about the order of modifiers (final, synchronized). These should be two PRs.